src/test/scala/outwatch/PerfTest.scala

```diff
@@ -0,0 +1,191 @@
+package outwatch
+
+import org.scalajs.dom.document
+import outwatch.dom._
+import outwatch.dom.dsl._
+
+import scala.scalajs.js
+
+class PerfTest extends JSDomSpec {
+
+  val numIterations = 100
+
+  "Perf" should "be" in {
```

Comment:
Benchmarks should not run when the tests run.
I suggest a sbt subproject `bench` where the benchmarks are called by the `main` method and can be run via `sbt bench/run`.